perf/index.js

```diff
@@ -0,0 +1,63 @@
+const levenshteinEditDistance = require('levenshtein-edit-distance');
+const fastLevenshtein = require('fast-levenshtein');
+const jsLevenshtein = require('js-levenshtein');
+const talisman = require('talisman/metrics/distance/levenshtein');
+const leven = require('leven');
+const jsLevenshteinEsm = require('esm')(module)('../');
+const {paragraphs, sentences, words} = require('esm')(module)('./data');
+
+if (!Object.entries) {
+  Object.entries = function( obj ) {
+    const ownProps = Object.keys( obj );
+    let i = ownProps.length;
+    const resArray = new Array(i); // preallocate the Array
+    while (i--) {
+      resArray[i] = [ownProps[i], obj[ownProps[i]]];
+    }
+    return resArray;
+  };
+}
+
+/**
+ * Executes a given function across the given dataset
+ *
+ * @param {string[]} data Dataset to execute against
+ * @return {Function} - bench runner
+ */
+function getBenchRunner(data) {
+  return (fn) => {
+    for (let i = 0; i + 1 < data.length; i += 2) {
+      const p1 = data[i];
+      const p2 = data[i + 1];
+      fn(p1, p2);
+    }
+  };
+}
+
+const paragraphBench = getBenchRunner(paragraphs);
+const sentenceBench = getBenchRunner(sentences);
+const wordBench = getBenchRunner(words);
+
+const suites = Object.entries({
```

Comment:
i'd rather shuffle this a bit so it looks something more like:

```ts
suite('50 paragraphs, etc', getBenchRunner(paragraphs));
suite('100 sentences, etc', getBenchRunner(sentences));
```

possibly where `getBenchRunner` has side-effects by calling `bench` its self (meaning no more loop at the bottom of this file).